src/main/java/com/csemosip/bookingservice/controller/Bookings.java

```diff
@@ -0,0 +1,62 @@
+package com.csemosip.bookingservice.controller;
+
+import com.csemosip.bookingservice.dto.BookingDTO;
+import com.csemosip.bookingservice.model.Booking;
+import com.csemosip.bookingservice.service.Impl.BookingServiceImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/bookings")
+public class Bookings extends AbstractController {
+
+    Logger log = LoggerFactory.getLogger(Bookings.class);
+    @Autowired
+    BookingServiceImpl bookingService;
+
+    @GetMapping
+    public ResponseEntity<Map<String, Object>> findBookings(
+            @RequestParam(name = "resourceId", required = false) Long resourceId,
```

Comment:
Change these params to `resource_id` and `date` (the argument names can stay the same)

Reply:
ok